applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -353,21 +355,22 @@ void BaseSolidElement::GetDofList(
 {
     KRATOS_TRY;
 
-    const SizeType number_of_nodes = GetGeometry().size();
-    const SizeType dimension = GetGeometry().WorkingSpaceDimension();
-    rElementalDofList.resize(0);
-    rElementalDofList.reserve(dimension*number_of_nodes);
+    const auto &r_geom = GetGeometry();
+    const SizeType number_of_nodes = r_geom.size();
+    const SizeType dimension = r_geom.WorkingSpaceDimension();
+    rElementalDofList.resize(0, false);
```

Comment:
I would resize if the size if not right and avoid push_back

Reply:
I agree with you but let's modify this in another PR (I'm afraid of breaking stuff)